functions/index.js

```diff
@@ -13,17 +15,37 @@ exports.addReport = functions.https.onRequest((req, res) => {
         message: 'Not allowed'
       });
     }
-
     const report = req.body;
 
-    const result = database.collection('reports').add(report)
+    return database.collection('reports').add(report)
       .then(ref => {
         return res.status(200).send('Success!');
       })
       .catch(error => {
         return res.status(500).send(`Error adding document: ${error}`);
       });
-    return result;
+  });
+});
+
+exports.getReport = functions.https.onRequest((req, res) => {
```

Comment:
Have you deployed this yet? If not, let us just set `initialQuery = collection` on line 60.

Reply:
It has been deployed.